channels/fr.m3u

```diff
@@ -347,3 +347,7 @@ https://streamer01.myvideoplace.tv/streamer02/hls/MATL_VLOC_PAD_100919_medium/in
 https://live.creacast.com/mirabelletv/smil:mirabelletv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="viaOccitanie.fr" tvg-name="viàOccitanie" tvg-country="FR" tvg-language="French" tvg-logo="https://viaoccitanie.tv/fileadmin/via_main/img/logos/viaOccitanie-header.png" group-title="Local",viàOccitanie
 https://streamer03.myvideoplace.tv/streamer02/hls/MDS_VIA_PAD_301117_medium/index.m3u8
+#EXTINF:-1 tvg-id="WéoNordPasdeCalais.fr" tvg-name="Wéo (Nord-Pas-de-Calais)" tvg-country="FR" tvg-language="French" tvg-logo="" group-title="Local",Wéo (Nord-Pas-de-Calais)
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/video/x62islg
+#EXTINF:-1 tvg-id="WéoNordPasdeCalais.fr" tvg-name="Wéo (Picardie)" tvg-country="FR" tvg-language="French" tvg-logo="" group-title="Local",Wéo (Picardie)
+https://query-strealkink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/video/x63085w
```

Comment:
Error: SSL connect error

Reply:
That's something weird... Dailymotion works, and the link is... just refused. That's an error that not on my scope, let me see.